practice_06.15.2019_generics/Task3_LinkedList/LinkedList.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using practice_06._15._2019_generics.Task1_MyList;
+
+namespace practice_06._15._2019_generics.Task3_LinkedList
+{
+    public class LinkedList<T> : IEnumerable<T>
```

Comment:
It looks like you copy-pasted this code from somewhere....

Reply:
There were no task to implement IEnumerable. You should have implemented only separate methods Add, Remove, Count.

Reply:
Yes, I asked not to watch this task yet, because I did not fully understand it